dev/wlp.lib.extract_fat/fat/src/wlp/lib/extract/PackageRunnableTest.java

```diff
@@ -297,7 +299,25 @@ private String executeTheJar(File extractDirectory, boolean useDummyUserDir, boo
             } else {
                 Log.info(c, "executeTheJar", "No messages.log - " + messagesLog.getAbsolutePath());
             }
+
```

Comment:
I suggest just using a BufferedReader with the `META-INF/MANIFEST.MF` JarEntry input stream to log it line by line.  I worry that the JarFile Manifest object may give us an empty  attributes even when the manifest resource has some corrupted data in it.

Reply:
Thanks Tom!  Updates made based on your suggestion.